src/garage/torch/algos/sac.py

```diff
@@ -0,0 +1,258 @@
+"""This modules creates a sac model in PyTorch."""
```

Comment:
Can you please make this more descriptive? I don't think it's a model.

You should probably include a citation for the paper you based this on, and the codebase if necessary.